src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,23 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeModal();
+    }
+
+    public void closeModal() {
+        WebElement closeButton = browser.driver.findElement(By.className("close"));
+        waitForElementToBeClickable(closeButton);
+        click(closeButton);
+        try {
+            Thread.sleep(1000);
```

Comment:
Looking at the code, it's likely this was copied over from `InstructorCourseEditPage`, which is fine.

Unfortunately, we're also trying to avoid timeouts like this (ref. #3317), so although I believe this shouldn't be done, the solution for this would probably require another PR to properly implement. The idea behind this new PR would be as follows:

1. Write a new method in `AppPage` for closing modals using the "x" button (we already have methods for "okay", "no", and "cancel", but none for the "x"). The `AppPage` class already has methods to wait for the visibility and disappearance of modals which could be reused.
2. Have tests such as this one and `InstructorCourseEditPageUiTest` (there may be others) use the new method instead.

However, the above plan also has an issue: the code for modals in the `AppPage` class only works for Bootbox modals, which are not used everywhere yet. Therefore, for the above to happen, all existing modals need to be migrated to Bootbox first.

I'm not sure if this is the best alternative however, so I'll have to defer the decision to @damithc.